modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

```diff
@@ -1410,6 +1415,77 @@ neo4j_home$ --nodes persons.csv --nodes games.csv --id-type string
 The `id` property of the nodes in the `persons` group will be stored as `long` type, while the `id` property of the nodes in the `games` group will be stored as `string` type, as the global `id-type` is a string.
 ====
 
+
+== Importing data that spans multiple lines
+
+The `--multiline-fields` option allows fields from an input source to span multiple lines, i.e. contain newline characters.
+For example:
+
+[source, shell, role=noplay]
+----
+bin/neo4j-admin database import full --nodes import/node_header.csv,import/node_data.csv --multiline-fields=true databasename
+----
+
+Where `import/node_data.csv` contains multiline fields, such as:
+
+[source, csv, role=nocopy]
+----
+id,name,birthDate,birthYear, birthLocation, description
+1, John, October 1st, 2000, New York, This is a multiline
+description
+----
+
+[NOTE]
+====
+Setting `--multiline-fields=true` can severely degrade the performance of the importer.
+Therefore, use it with care, especially with large imports.
+====
+
+Starting from 5.26, the `--multiline-fields` option can be used in conjunction with the `--multiline-fields-format` option, which controls the parsing of the input source.
+The default value `v1` uses the current processing method for multiline fields.
+Option `v2` allows you to specify a list of files (regular expressions allowed) that contain multiline fields and are processed much more performant, with the restriction that text fields must be quoted.
```

Comment:
is performantly a word? `and are processed much more performant` doesn't scan right

Reply:
Good catch!

Reply:
There is such a word, but I'll write it in a different way. Thanks.